nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -457,198 +153,322 @@ def forward(self, obj):
     # get the name of the attribute, for example
     # prim::GetAttr[name="module_list"](%self.1)
     assert node.kind() == 'prim::GetAttr'
-    pattern = '\[name=\"(.*?)\"\]'
+    pattern = '\\[name=\"(.*?)\"\\]'
```

Comment:
why double `\` here?

Reply:
pylint report a warning in another line in which the single '\' with the next character will be treated as an escape character, so i doubled all the single '\'.